src/components/displayDocuments/documents.tsx

```diff
@@ -0,0 +1,39 @@
+export interface Document {
+  title: string;
+  url: string;
+  imageName: string;
```

Comment:
I'm thinking if you want to pass in the full image src (and name it `imageSrc`) into this instead of a image name.This will allow us to render image that are not even hosted on our repo later.

But it seemed like this is not part of this PR, so maybe you can create a separate issue and address that later.